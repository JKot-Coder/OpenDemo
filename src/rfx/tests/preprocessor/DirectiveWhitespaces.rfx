// :RUN_RFXC -P %STD_OUTPUT% %INPUT%
// Test odd whitespaces combinations in directives
#define FOO_1
#       define FOO_2
#/**/define FOO_3
#\
define FOO_4
#/**/\
define/**/\
BAR SUCCESS//This is skipped
#//empty\
directive

#if defined(FOO_1) && defined(FOO_2)  && defined(FOO_3) && defined(FOO_4)
   BAR 
#endif