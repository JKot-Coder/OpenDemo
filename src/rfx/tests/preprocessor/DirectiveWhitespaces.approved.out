SourceOutput:[
    #line 15 "../src/rfx/tests/preprocessor/DirectiveWhitespaces.rfx"
    SUCCESS
]
Diagnostic:[
]
